src/bindings/interface/model.i

```diff
@@ -76,6 +76,25 @@ determine the full path to the source model relative to this one.";
 %feature("docstring") libcellml::Model::fixVariableInterfaces
 "Fix variable interfaces throughout the model.";
 
+%feature("docstring") libcellml::Model::addImportSource
+"Manually add an ImportSource to the Model.";
```

Comment:
"import source" and "model", not "ImportSource" and "Model". Same below.

Reply:
TBD in #618